quack_app/test/golden_widget_test.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:quack_app/main.dart';
+import 'package:golden_toolkit/golden_toolkit.dart';
+
+void main() {
+  var test1Name = 'Homepage';
```

Comment:
nit: please name this variable something more descriptive like: `homepage_test`.

Everything else looks good! Thanks for your work on this!

Reply:
Sorry, I should have specified - I meant the variable name